CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ----------------------
 ### Added
 - **Support:** GIMP (`.gpl`)
+- **Support:** Minecraft Function (`.mcfunction`)
```

Comment:
Please merge this into one line:

~~~markdown
- **Support:** GIMP (`.gpl`), Minecraft (`.mcfunction`, `pack.mcmeta`, `.png.mcmeta`)
~~~

The change-log is currently 1421 lines long, and that's with all relevant changes grouped on the same line. Imagine how long the file would become if we split it across lines...